src/router.js

```diff
@@ -46,6 +47,17 @@ var Router = {
       }
     });
 
+    try {
+      let rulesEngine = require('rules-engine/index.js');
+      // Issue a token without an associated user
+      JSONWebToken.issueToken(-1).then(jwt => {
```

Comment:
When a user is removed all of the JWTs associated with their user ID are deleted from the database. When a user logs out only the JWT associated with that session is deleted.

By not being associated with any user this JWT is effectively permanent unless it is manually revoked which is a bit problematic. If we want to use this pattern in the future it would be good to have expiring tokens and to delete all tokens with a user id of -1 when starting the server